tests/unit/suites/mfx_dispatch/linux/CMakeLists.txt

```diff
@@ -78,4 +87,7 @@ if(TARGET gmock)
   endif()
 endif()
 
-set_property(TEST run_mfx_dispatch_test PROPERTY ENVIRONMENT "LD_LIBRARY_PATH=${LIBRARY_PATH}")
+set_property(TEST run_mfx_dispatch_test
```

Comment:
Well, I do not have a preference on this vs. prev. coding style and on the location of the gtest and/or tests themselves. However, moving single tiny test around without adding new stuff hardly makes any sense...

Reply:
@dvrogozh,

I've tried to somehow align style over that certain file since I've touched it, because currently we have a mess, that was an intention. I will revert these changes If u insist.